easybuild/tools/toolchain/toolchain.py

```diff
@@ -727,6 +729,15 @@ def prepare_compiler_cache(self, cache_tool):
         self.cached_compilers.update(compilers)
         self.log.debug("Cached compilers (after preparing for %s): %s", cache_tool, self.cached_compilers)
 
+    @staticmethod
+    def is_rpath_wrapper(path):
+        """
+        Check whether command at specified location already is an RPATH wrapper script rather than the actual command
+        """
+        in_rpath_wrappers_dir = os.path.basename(os.path.dirname(path)) == RPATH_WRAPPERS_SUBDIR
+        calls_rpath_args = 'rpath_args.py $CMD' in read_file(path)
```

Comment:
The RPATH wrapper script that is in place is a templated version of `easybuild/scripts/rpath_wrapper_template.sh.in`. That doesn't mean we can't hash it, but I'd rather have a stupid yet safe check in place to really avoid wrapping an existing wrapper. We really *really* REALLY don't want to include a fork bomb in EasyBuild...

Checking a hash is really strict, you just need a single byte changed somehow to have a hash mismatch...